modules/openy_gc_storage/config/install/field.storage.eventseries.field_ls_image.yml

```diff
@@ -0,0 +1,19 @@
+langcode: en
+status: true
+dependencies:
+  module:
+    - media
+    - recurring_events
+id: eventseries.field_ls_image
+field_name: field_ls_image
+entity_type: eventseries
+type: entity_reference
+settings:
+  target_type: media
```

Comment:
We need to limit it to Image bundle only

Reply:
Limited here - https://github.com/fivejars/openy_gated_content/pull/30/files#diff-a197f9ec7a697ac41c1c395bc70f6703R28